lib/reporter.ts

```diff
@@ -326,7 +330,7 @@ class CucumberJsJsonReporter extends WDIOReporter {
     public getCurrentStep(): Step {
         const currentScenario = this.getCurrentScenario();
 
-        return currentScenario.steps[currentScenario.steps.length - 1];
+        return currentScenario.steps![currentScenario.steps!.length - 1];
```

Comment:
What happens if `.step` is undefined?

Reply:
Code updated to check if step is undefined